libavcodec/vaapi_decode.h

```diff
@@ -45,6 +45,10 @@ typedef struct VAAPIDecodePicture {
     int                nb_slices;
     VABufferID           *slice_buffers;
     int                   slices_allocated;
+
+    VASurfaceID           sfc_surface;
```

Comment:
better use a list for this. In the feature we may support 1:N

Reply:
The name(sfc) may need change

Reply:
Current code only one sub frame, we can extend it into list when we support multi sub frame.